### PR TITLE
商品購入確認ページ（仁井田）

### PR DESCRIPTION
**Pull request by Atsunori-Niida**
WHAT
商品購入ページのビューを作成

### CODE REVIEW

File: app/assets/stylesheets/create.scss [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+    .header-create-img{
+      text-align: center;
+      line-height: 160px;
+    }
+  }
+  .body-create{
+    width: 700px;
+    margin: 0 auto;
+    background: #f5f5f5;
+    .box_1{
```
Comment (TOMARUMARU) [inline app/assets/stylesheets/create.scss]: 1とか6とかありますが、classの命名きちんとしましょう

**Comment by TOMARUMARU** [comment]
conflict解消しましょう

File: app/assets/stylesheets/create.scss [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    .box_item{
+      height: 145px;
+      background: #fff;
+      border-top: 1px solid #f5f5f5;
+      .box_item_sep{
+      max-width: 343px;
+      margin: 0 auto;
+      padding-top: 30px;
+      display: flex;
+      .box_item_sep--a{
```
Comment (shohei-shimoda) [inline app/assets/stylesheets/create.scss]: > __[stylelint]__ Unexpected empty block ([see on Sider](https://sider.review/gh/repos/228985888/pulls/7#issue-4995229))
>
> <details>
>
> | Rule | Severity |
> | - | - |
> | block-no-empty | error |
>
> References:
> - <https://github.com/stylelint/stylelint/tree/12.0.0/lib/rules/block-no-empty>
>
> </details>

アロハシャツ

File: app/assets/stylesheets/create.scss [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+      .box_item_detail{
+        margin-left: 16px;
+        font-size: 14px;
+
+      .box_item_detail-2{
+        display: flex;
+
+        .box_item_detail--b{
+        font-size: 13px;
+        font: sans-serif;
```
Comment (shohei-shimoda) [inline app/assets/stylesheets/create.scss]: > __[stylelint]__ Unexpected shorthand "font" after "font-size" ([see on Sider](https://sider.review/gh/repos/228985888/pulls/7#issue-4995230))
>
> <details>
>
> | Rule | Severity |
> | - | - |
> | declaration-block-no-shorthand-property-overrides | error |
>
> References:
> - <https://github.com/stylelint/stylelint/tree/12.0.0/lib/rules/declaration-block-no-shorthand-property-overrides>
>
> </details>

font-sizeに書き換えて欲しいみたいです。

File: app/assets/stylesheets/create.scss [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+        font-weight: bold;
+        }
+
+
+        .box_item_detail--c{
+          padding-left: 8px;
+          font-size: 12px;
+          font: sans-serif;
```
Comment (shohei-shimoda) [inline app/assets/stylesheets/create.scss]: > __[stylelint]__ Unexpected shorthand "font" after "font-size" ([see on Sider](https://sider.review/gh/repos/228985888/pulls/7#issue-4995231))
>
> <details>
>
> | Rule | Severity |
> | - | - |
> | declaration-block-no-shorthand-property-overrides | error |
>
> References:
> - <https://github.com/stylelint/stylelint/tree/12.0.0/lib/rules/declaration-block-no-shorthand-property-overrides>
>
> </details>

font-size

File: app/assets/stylesheets/create.scss [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+       margin: 0 auto;
+       padding-top: 32px;
+      }
+      .box_money_a{
+      display: flex;
+      height: 30px;
+      .box_money_a-1{
+        width: 161px;
+        font-size: 18px;
+        font: sans-serif;
```
Comment (shohei-shimoda) [inline app/assets/stylesheets/create.scss]: > __[stylelint]__ Unexpected shorthand "font" after "font-size" ([see on Sider](https://sider.review/gh/repos/228985888/pulls/7#issue-4995232))
>
> <details>
>
> | Rule | Severity |
> | - | - |
> | declaration-block-no-shorthand-property-overrides | error |
>
> References:
> - <https://github.com/stylelint/stylelint/tree/12.0.0/lib/rules/declaration-block-no-shorthand-property-overrides>
>
> </details>

font-type

**Pull request merged**